doc/articles/debugging-uno-ui.md

```diff
@@ -25,6 +25,12 @@ step into Uno.UI without downloading the repository.
 Make sure **Enable source link support** check box is checked in **Tools** / **Options**
 / **Debugging** / **General** properties page.
 
+## Updating the Nuget packages used by the Uno.UI solution
```

Comment:
That is something we could and should automate with our nuget updater ? @benventive  What do you think?

Reply:
The nuget updater should be picking those up, I don't think it's filtering on the `Include` attribute.

Reply:
Will be fixed in nuget updater.